### PR TITLE
Add script for generating differential coverage reports

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

File: infra/base-images/base-runner/generate_differential_cov_report.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+#
```
Comment (jonathanmetzman) [inline infra/base-images/base-runner/generate_differential_cov_report.py:2]: TODO: Run Max's scripts to prettify the report.

**Comment by jonathanmetzman** [comment]
TBR. I think this is pretty low risk

**Review by Alan32Liu — COMMENTED** [review]
LGTM

**Pull request merged**